COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -1657,13 +1677,13 @@ If desired, you can also easily submit a job, wait for it to complete, and downl
 zowe jobs submit ds "Z80462.JCL(HELLO)" -d .
```

Comment:
There are 10 occurrences of this ID being mentioned on this Lab. Maybe we can change it to Z99998?